### ISSUE TITLE
Documentation issue, Link on website incorrect

### ISSUE THREAD

**Issue by wvdvegt** [esi/esi-issues#1260]
# Bug

https://developers.eveonline.com/blog/article/esi-best-practices-using-underscore-routes

The link https://esi.tech.ccp.is/_latest/swagger.json at the bottom seems outdated.
Should point to https://esi.evetech.net/ instead.

## Request

https://esi.tech.ccp.is/_latest/swagger.json

### Status Code

`404`

## Expected

https://esi.evetech.net/_latest/swagger.json

is the correct link and returns json.

# Checklist

Check all boxes that apply to this issue:

- [x] Bug description is provided
- [x] Request path is provided
- [x] Response status code is provided
- [ ] Response headers are provided
- [ ] Response body is provided
- [x] Expected response is provided

**Comment by Blacksmoke16** [comment]
tl;dr those blogs were written before ESI was using the current `esi.evetech.net` domain.  Probably wouldn't hurt to update them to the newer URL.

Also not really sure where issues like this should go, but I just threw a `documentation` label on it...prob is good enough.

**Comment by GoldenGnu** [comment]
The `_` (versioned routes) are accessible in the web ui (https://esi.evetech.net) by clicking the 📌 icon next to legacy/latest/dev

**Comment by Xamber** [comment]
Internal issue: EO-11706
